app/src/main/java/com/mensinator/app/settings/SettingsScreen.kt

```diff
@@ -443,6 +451,67 @@ private fun ColorPickerPreview() {
     }
 }
 
+@Composable
+private fun SettingText(
+    text: String,
+    modifier: Modifier = Modifier,
+    dbKey: String,
+) {
+    val dbHelper: IPeriodDatabaseHelper = koinInject()
```

Comment:
Please don't use dbHelper inside here. Instead, add a property to `SettingsViewModel.ViewState` with the (initial) value of the message.

This composable should then create a mutable var which gets initialized with the message from the ViewState.
Once the user Taps the save button, call a method from the ViewModel to save this setting (for that, add an `onSave: () -> Unit` parameter to this composable).

Reply:
Please check now and see if I've done it properly

Reply:
Yep, awesome